setup.py

```diff
@@ -23,8 +23,8 @@
     author="City of Helsinki",
     author_email="dev@hel.fi",
     install_requires=[
-        "Django<3.0",
-        "django-parler<2.0",
+        "Django",
+        "django-parler",
```

Comment:
I remember there was a reason to lock django-parler at version 1.9
Check this out: https://github.com/City-of-Helsinki/django-ilmoitin/pull/8/files

Reply:
Should we then add the migration and require django-parler>=2.0 ?

Reply:
The changes on the migration are not breaking and we could just add it and upgrade it.
```diff
- master = models.ForeignKey(
+ master = parler.fields.TranslationsForeignKey(
    editable=False,
    null=True,
    on_delete=django.db.models.deletion.CASCADE,
    related_name="translations",
    to="django_ilmoitin.NotificationTemplate",
)
```

Reply:
Sound good, in that case I think we still need to require `django-parler>=2.0` since `parler.fields.TranslationsForeignKey` was part of the 2.0 release of django-parler.

Reply:
yes, now that you have added the `parler.fields.TranslationsForeignKey`, we need to pin `django-parler>=2.0`

Reply:
Yup yup, sounds good. 👍